SU-HSDO/suhumsci
# STN-688 : Refine Captions/Credits for Vertical Linked Card

# [STN-688](https://sparkbox.atlassian.net/browse/STN-688) READY FOR REVIEW

## Summary
The work in this PR updates the spacing for captions/credits on vertical linked cards to allow all of the caption/credit content to be visible.

## Need Review By (Date)
3/2

## Urgency
low

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. Go to the following pages which have vertical linked cards presented in a variety of ways and confirm that the caption/credit text on vertical linked cards is fully visible:
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/texts-cards-row-wells
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/texts-cards-row-wells-hb-raised-cards
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/texts-cards-row-wells-hb-stretch-vertical-linked-cards
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/qa-postcards
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/qa-raised-postcards
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/qa-stretch-vertical-linked-postcards 

### If you do not see the caption/credits on vertical linked cards, here's some debugging hints:
Check to see if you have updated your configuration for images to display them. Go to Structure > Media Types > Image. 

Click on the `Manage form display` tab. If the `Caption/Credit` field is located in the `Disabled` section, move it up beneath the `Image` field to enable it. Save.

![manage form display](https://user-images.githubusercontent.com/12678977/109194227-389bbf00-7767-11eb-892d-47f2b7a2ff5d.png)

Click on the `Manage display` tab. If the `Caption/Credit` field is located in the `Disabled` section, move it up beneath the `Image` field to enable it. Update the `Label` value to `- Hidden -` and the `Format` value to `Smart trimmed`. Click on the gear icon to update the smart trimmed value to 160 characters. Save.

![manage display](https://user-images.githubusercontent.com/12678977/109194644-a1833700-7767-11eb-969d-bc0a66555cb8.png)
